common.inc.bat

```diff
@@ -30,11 +30,11 @@ if "%BUILD_ARCH%" == "Win32" (
 		set CMAKE_GENERATOR=Visual Studio 16 2019
 	)
 	set CMAKE_GENERATOR_PLATFORM=Win32
-    set QT_BIN_PATH=%QT_PATH%\msvc%VS_VERSION%\bin
-	set QT_PREFIX=%QT_PATH%\msvc%VS_VERSION%
+    set QT_BIN_PATH=%QT_PATH%\msvc%QT_VS_VERSION%\bin
```

Comment:
@FlexW `QT_VS_VERSION` is set to **2019** similar to `VS_VERSION`. What does it change, then?

Reply:
In stable-3.3 I needed to change it to different values. I think it makes sense because that two variables were just by accident the same. They can differ.

Reply:
for example in stable-3.3 VS_VERSION is 2019 and QT_VS_VERSION is 2017